File: LOL-pick-and-ban/ClientApp/src/app/Models/PickAndBan.ts
import { Misc } from '../Components/draft-kingdom-main/misc'
import { Inject } from '@angular/core'
export class PickAndBan {
  sessID: string
  red_pick: PickAndBanStyles[]
  red_ban: PickAndBanStyles[]
  blue_pick: PickAndBanStyles[]
  blue_ban: PickAndBanStyles[]
  constructor(sessID:string) {
    this.sessID = sessID
    this.red_pick = [new PickAndBanStyles(), new PickAndBanStyles(), new PickAndBanStyles()
      , new PickAndBanStyles(), new PickAndBanStyles()]
    this.red_ban = [new PickAndBanStyles(), new PickAndBanStyles(), new PickAndBanStyles()
      , new PickAndBanStyles(), new PickAndBanStyles()]
    this.blue_pick = [new PickAndBanStyles(), new PickAndBanStyles(), new PickAndBanStyles()
      , new PickAndBanStyles(), new PickAndBanStyles()]
    this.blue_ban = [new PickAndBanStyles(), new PickAndBanStyles(), new PickAndBanStyles()
      , new PickAndBanStyles(), new PickAndBanStyles()]
  }
}
export class PickAndBanStyles {
  img_src: string
  data_source_index: string
  constructor() {
    this.img_src = new Misc().original_pre_pick_src
    this.data_source_index = "null";
  }
}

File: LOL-pick-and-ban/Models/PickAndBan.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace LOL_pick_and_ban.Models
{
    public class PickAndBan
    {
        public string sessID { get; set; }
        public PickAndBanStyles[] red_pick { get; set; }
        public PickAndBanStyles[] blue_pick { get; set; }
        public PickAndBanStyles[] red_ban { get; set; }
        public PickAndBanStyles[] blue_ban { get; set; }
        public PickAndBan(string sessID)
        {
            this.sessID = sessID;
            red_pick = new PickAndBanStyles[5] { new PickAndBanStyles(), new PickAndBanStyles(), new PickAndBanStyles(),
                                                    new PickAndBanStyles(),new PickAndBanStyles(),};
            red_ban = new PickAndBanStyles[5] { new PickAndBanStyles(), new PickAndBanStyles(), new PickAndBanStyles(),
                                                    new PickAndBanStyles(),new PickAndBanStyles(),};
            blue_pick = new PickAndBanStyles[5] { new PickAndBanStyles(), new PickAndBanStyles(), new PickAndBanStyles(),
                                                    new PickAndBanStyles(),new PickAndBanStyles(),};
            blue_ban = new PickAndBanStyles[5] { new PickAndBanStyles(), new PickAndBanStyles(), new PickAndBanStyles(),
                                                    new PickAndBanStyles(),new PickAndBanStyles(),};
        }
    }
    public class PickAndBanStyles
    {
        public string img_src { get; set; }
        public string data_source_index { get; set; }
        public PickAndBanStyles()
        {
            img_src = "../assets/icon/question-mark.svg";
            data_source_index = "null";
        }
    }
}
